### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by ekaneva**
# Add runbook

## What

Add a runbook to this service to improve the general SOS score.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1181)

## Anything, in particular, you'd like to highlight to reviewers

Nope, nothing special. :)

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by bboykov — DISMISSED** [review]

**Review by rnov — DISMISSED** [review]

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+PartiallyAutomated
+
+## Rollback Process Type
+
+Manual
+
+## Release Details
+
+Manual failover is **not required** when a new version of the service is deployed to production. Releasing of no mapper *should not trigger* an automated failover. But if in doubt, do a faiolver.
```
Comment (karsov) [inline runbooks/runbook.md]: "Releasing of no mapper"?

The information here is not correct. Unfortunately the "methode-article-mapper" is coupled with the "read" healthcheck category so a release albeit short in time, may cause a failover.

**Review by karsov — CHANGES_REQUESTED** [review]
Almost all looks perfect but a change in the failover description is needed. You may also update all links to be surrounded by "<>" according to the Markdown standard that is agreed upon.

**Comment by karsov** [comment]
Also we have lost some of the information present in the old runbook:

> Methode Article Mapper reads from the NativeCMSPublicationEvents queue, identifies the articles received from Methode, transforms them into UP format and writes them to the CMSPublicationEvents queue.

File: runbooks/runbook.md (hunk 1 above)
Comment (ekaneva) [reply runbooks/runbook.md]: Thanks for the sharp eye on this, @karsov The failover instructions are altered.

**Review by karsov — APPROVED** [review]

**Review by bboykov — APPROVED** [review]

**Pull request merged**